docs/source/contribution/how_to_write_docs.rst

```diff
@@ -0,0 +1,76 @@
+.. _how_to_write_docs:
+
+Contribute to docs
+==================
+
+How to build the docs locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To build the docs, clone the repository
+::
+
+   $ git clone https://github.com/pypr/pysph
+
+Make sure to work in an ``pysph`` environment. I will proceed with the further
+instructions assuming that the repository is cloned in home directory. Change to
+the ``docs`` directory and run ``make html``. ::
+
+   $ cd ~/pysph/docs/
+   $ make html
+
+
+Possible error one might get is
+::
+
+   $ sphinx-build: Command not found
+
+Which means you don't a have `sphinx-build` in your system. To install across the system do
+::
+
+   $ sudo apt-get install python3-sphinx
```

Comment:
Please also mention the pip instruction for those without apt access or on a different platform.